### PR TITLE
VP-4566 Fix: add validation for page creating operation

### PR DESCRIPTION
**Pull request by Vectorfield4**
https://virtocommerce.atlassian.net/browse/VP-4566

### Proposed of changes
Describe the technical details of realization provided by you.

### Make sure these boxes are checked:
- [x] Check all the changes in github PR - files count (non of them are redundant, have meaningful changes, all are added), if target branch is correct
- [x] Check methods and variable namings - it should be self descriptive, no typos
- [x] Check you did not introduce breaking changes in API and public models/services.
- [x] Respect extensibility - https://community.virtocommerce.com/t/extensibility-basics-the-domain-model-and-persistence-layer-extension/141
- [x] Follow [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and [SOLID](https://en.wikipedia.org/wiki/SOLID) principles
- [x] For unit tests - follow Microsoft best practices: https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices
- [x] Consolidate solution dependencies in case you are using newer version, update VC module dependencies in module.manifest. Do not upgrade 3rd party packages that are shipped with the platform with the version newer than in the platform.
- [x] Check code style conventions - https://github.com/VirtoCommerce/styleguide/blob/master/csharp.md
- [x] Check PR have a concise and descriptive title, follow [git commit message rules](https://github.com/VirtoCommerce/styleguide/blob/master/gitcommits.md)

### CODE REVIEW

**Comment by Vectorfield4** [comment]
Merge with https://github.com/VirtoCommerce/vc-platform/pull/2009

File: src/VirtoCommerce.ContentModule.Web/Controllers/Api/ContentController.cs [hunk 1]
```diff
@@ -245,8 +246,16 @@ public async Task<ActionResult> Unpack(string contentType, string storeId, [From
         [Route("folder")]
         [Authorize(Permissions.Create)]
         [ProducesResponseType(typeof(void), StatusCodes.Status204NoContent)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
```
Comment (akak1977) [inline src/VirtoCommerce.ContentModule.Web/Controllers/Api/ContentController.cs:249]: May be 
`[ProducesResponseType(typeof(string), StatusCodes.Status400BadRequest)] `
?

Comment (Vectorfield4) [reply src/VirtoCommerce.ContentModule.Web/Controllers/Api/ContentController.cs:249]: Nope, we return array of error objects.

Comment (akak1977) [reply src/VirtoCommerce.ContentModule.Web/Controllers/Api/ContentController.cs:249]: Ok

**Review by yecli — APPROVED** [review]

**Pull request merged**